config/locales/hyrax.zh.yml

```diff
@@ -9,12 +9,7 @@ zh:
       messages:
         conflict: 您的更改无法保存，因为另一用户(或后台作业）在您开始编辑后更新了%{model}。 请确保所有文件附件已成功完成之后再试一次。这已经是最新刷新后保存的副本%{model}
   blacklight:
-    application_name: Blacklight
```

Comment:
Changes to `zh.blacklight` keys were done by the `i18n-tasks` tool, and this will keep happening until Hyrax uses a version of Blacklight that includes a Chinese translation. I have backported a Blacklight commit to its 6.x release series so that we don't have to keep manually restoring these keys: https://github.com/projectblacklight/blacklight/pull/1823

Reply:
Heads-up, @jcoyne

Reply:
I am happy to bump the version of blacklight in addition to the changes in this PR once https://github.com/projectblacklight/blacklight/pull/1823 is merged and a new 6.x release is available.